rules/windows/registry_event/sysmon_modify_screensaver_binary_path.yml

```diff
@@ -0,0 +1,29 @@
+title: Path To Screensaver Binary Modified
+id: 67a6c006-3fbe-46a7-9074-2ba3b82c3000
+status: experimental
+description: >
+  Detects value modification of registry key containing path to binary used as screensaver. 
+  Adversaries may establish persistence by executing malicious content triggered by user inactivity. 
+  Screensavers are programs that execute after a configurable time of user inactivity and consist of Portable Executable (PE) files with a .scr file extension.
+references:
+  - https://attack.mitre.org/techniques/T1546/002/
+  - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1546.002/T1546.002.md
+  - https://www.welivesecurity.com/wp-content/uploads/2017/08/eset-gazer.pdf
+tags:
+  - attack.persistence
+  - attack.privilege_escalation
+  - attack.t1546.002
+author: Bartlomiej Czyz @bczyz1, oscd.community
+date: 2020/10/11
+logsource:
+  category: registry_event
+  product: windows
+detection:
+  selection:
+    TargetObject|endswith: '\Control Panel\Desktop\SCRNSAVE.EXE' # HKEY_CURRENT_USER\Control Panel\Desktop\SCRNSAVE.EXE
+  filter:
+    ParentImage: 'C:\Windows\System32\services.exe'
```

Comment:
change filter  to:
```
filter:
Image|endswith: 
	- '\rundll32.exe'
	- '\explorer.exe'
```
because events of category _registry_event_ do not contain the _ParentImage_ field.

Everything else looks good!

Reply:
my oversight, thanks for pointing out!
fixed in 8ae42bca7ceb3f06bc73b117f1ea48e11f06d803